services/server/config/config.go

```diff
@@ -251,6 +274,32 @@ func loadConfigFile(path string) (*Config, error) {
 	return config, nil
 }
 
+func validaConfigUnsupported(path string) {
+	var tmpCfg Config
+	cfgFile, err := os.Open(path)
+	if err != nil {
+		log.L.Errorf("failed to open config file,err: %v", err)
+		return
+	}
+	decoder := toml.NewDecoder(cfgFile).Strict(true)
+	if err = decoder.Decode(&tmpCfg); err != nil {
+		v := reflect.ValueOf(*decoder)
+		field := v.FieldByName("visitor").FieldByName("keys")
```

Comment:
go-toml v2 seems to have better support for missing errors, we can add a comment to remind us to update here as well after upgrading go-toml v2

Reply:
We may not need to upgrade to v2 now
If other maintainers agree to use reflection in this way, I think it would be useful to add an additional optimization note for future upgrades to go-toml v2